src/account.rs

```diff
@@ -0,0 +1,589 @@
+#![allow(dead_code)]
+
+use crate::*;
+
+#[account(zero_copy)]
+#[repr(packed)]
+pub struct State {
+    // Admin authority
+    pub admin: Pubkey,                                   // 32
+    pub state_nonce: u8,                                 // 1
+    pub serum_nonce: u8,                                 // 1
+    pub mint_auth_nonce: u8,                             // 1
+    pub num_underlyings: u8,                             // 1
+    pub _null: u64,                                      // 8
+    pub strike_initialization_threshold_seconds: u32,    // 4
+    pub pricing_frequency_seconds: u32,                  // 4
+    pub liquidator_liquidation_percentage: u32,          // 4
+    pub insurance_vault_liquidation_percentage: u32,     // 4
+    pub native_d1_trade_fee_percentage: u64,             // 8
+    pub native_d1_underlying_fee_percentage: u64,        // 8
+    pub native_whitelist_underlying_fee_percentage: u64, // 8
+    pub native_deposit_limit: u64,                       // 8
+    pub expiration_threshold_seconds: u32,               // 4
+    pub position_movement_fee_bps: u8,                   // 1
+    pub margin_concession_percentage: u8,                // 1
+    pub treasury_wallet_nonce: u8,                       // 1
+    pub native_option_trade_fee_percentage: u64,         // 8
+    pub native_option_underlying_fee_percentage: u64,    // 8
+    pub referrals_admin: Pubkey,                         // 32
+    pub referrals_rewards_wallet_nonce: u8,              // 1
+    pub _padding: [u8; 107],                             // 107
+} // 255
+
+#[account(zero_copy)]
+#[repr(packed)]
+pub struct ZetaGroup {
+    pub nonce: u8,                                // 1
+    pub vault_nonce: u8,                          // 1
+    pub insurance_vault_nonce: u8,                // 1
+    pub front_expiry_index: u8,                   // 1
+    pub halt_state: HaltState,                    // 167
+    pub underlying_mint: Pubkey,                  // 32
+    pub oracle: Pubkey,                           // 32
+    pub greeks: Pubkey,                           // 32
+    pub pricing_parameters: PricingParameters,    // 112
+    pub margin_parameters: MarginParameters,      // 120
+    pub products: [Product; 46],                  // 137 * 43 = 5891
+    pub products_padding: [Product; 91],          //
+    pub perp: Product,                            // 43
+    pub expiry_series: [ExpirySeries; 2],         // 32 * 6 = 192
+    pub expiry_series_padding: [ExpirySeries; 4], //
+    pub total_insurance_vault_deposits: u64,      // 8
+    pub asset: Asset,                             // 1
+    pub expiry_interval_seconds: u32,             // 4
+    pub new_expiry_threshold_seconds: u32,        // 4
+    pub perp_parameters: PerpParameters,          // 24
+    pub perp_sync_queue: Pubkey,                  // 32
+    pub oracle_backup_feed: Pubkey,               // 32
+    pub padding: [u8; 966],                       // 966
```

Comment:
Could this be updated to the latest - it's slightly out of date, theres a few fields like 

perpOnly, flexUnderlying

Reply:
cheers